### DISCUSSION TITLE
Please add 20 slots bag to Mythic point exchange list

### DISCUSSION THREAD

**Discussion by BaoZaiz** [chromiecraft/chromiecraft#6308] [Suggestions]
Please add 20 slots bag (loot from Magtheridon) into Mythic point exchange list.

There are players with 10 points who don't need gears from Gruul / Mag raid, except for 20 slots bag, especially those who unlucky who kept lost in dice roll

Thank you

**Comment by xinmpg** [comment]
I concur with this sentiment
